### ISSUE TITLE
Moonraker Warning(s) on start up, everything else working properly.

### ISSUE THREAD

**Issue by VashBaldeus** [TheSpaghettiDetective/moonraker-obico#55]
The warnings:

> **Moonraker warning**
> 
> Unparsed config option 'managed_services:
> moonraker-obico' detected in section [update_manager moonraker-obico]. This may be an option no longer available or could be the result of a module that failed to load. In the future this will result in a startup error.
> 
> **Moonraker warning**
> 
> Unparsed config option 'install_script: install.sh' detected in section [update_manager moonraker-obico]. This may be an option no longer available or could be the result of a module that failed to load. In the future this will result in a startup error.
> 
> **Moonraker warning**
> 
> Unparsed config option 'requirements: requirements.txt' detected in section [update_manager moonraker-obico]. This may be an option no longer available or could be the result of a module that failed to load. In the future this will result in a startup error.
> 
> **Moonraker warning**
> 
> Unparsed config option 'origin: https://github.com/TheSpaghettiDetective/moonraker-obico.git' detected in section [update_manager moonraker-obico]. This may be an option no longer available or could be the result of a module that failed to load. In the future this will result in a startup error.
> 
> **Moonraker warning**
> 
> [update_manager]: Failed to load extension moonraker-obico: [Errno 13] Permission denied: '/root/moonraker-obico-env/bin/pip'

Regardless of these issues, Obico connects to my self-hosted Obico server and works normally.

**Comment by zellneralex** [comment]
Can you share your moonraker.log, that looks more like an config error

**Comment by VashBaldeus** [comment]
> Can you share your moonraker.log, that looks more like an config error

https://pastebin.com/ziNhRZyP

**Comment by zellneralex** [comment]
ok that is not normal 
```
[update_manager moonraker-obico]
type: git_repo
path: ~/moonraker-obico
origin: https://github.com/TheSpaghettiDetective/moonraker-obico.git
env: /root/moonraker-obico-env/bin/python
requirements: requirements.txt
install_script: install.sh
managed_services:	moonraker-obico
```
Did you try to install obico as root?

**Comment by zellneralex** [comment]
I would expect it would look like 
```
[update_manager moonraker-obico]
type: git_repo
path: ~/moonraker-obico
origin: https://github.com/TheSpaghettiDetective/moonraker-obico.git
env: ~/moonraker-obico-env/bin/python
requirements: requirements.txt
install_script: install.sh
managed_services: moonraker-obico
```
But for that I need to see your file system

**Comment by kennethjiang** [comment]
@zellneralex Good catch! I think it's because the user installed moonraker-obico as root. @VashBaldeus Please confirm if you can see this message.

**Comment by VashBaldeus** [comment]
> @zellneralex Good catch! I think it's because the user installed moonraker-obico as root. @VashBaldeus Please confirm if you can see this message.

**Comment by kennethjiang** [comment]
@VashBaldeus Can you confirm if you installed moonraker-obico as root?

**Comment by VashBaldeus** [comment]
> @VashBaldeus Can you confirm if you installed moonraker-obico as root?
I installed it as `pi` user on my raspberry pi. when running the install command I applied `sudo`, could this be the cause of the issue? if so, how do i re-install?

**Comment by kennethjiang** [comment]
> > @VashBaldeus Can you confirm if you installed moonraker-obico as root?
> > I installed it as `pi` user on my raspberry pi. when running the install command I applied `sudo`, could this be the cause of the issue? if so, how do i re-install?

This could be the issue. Yes please re-install without `sudo`. The script will run sudo to elevate the access when/if necessary.

**Comment by VashBaldeus** [comment]
> > > @VashBaldeus Can you confirm if you installed moonraker-obico as root?
> > > I installed it as `pi` user on my raspberry pi. when running the install command I applied `sudo`, could this be the cause of the issue? if so, how do i re-install?
> 
> This could be the issue. Yes please re-install without `sudo`. The script will run sudo to elevate the access when/if necessary.

Okay, wish I knew how to uninstall it, since i am not super linux savvy.
